src/main/java/com/renatusnetwork/momentum/data/plots/Plot.java

```diff
@@ -77,8 +77,10 @@ public boolean canBuild(Player player)
     public boolean isTrusted(String uuid) { return trustedUUIDs.contains(uuid); }
 
     public void addTrusted(PlayerStats playerStats) { trustedUUIDs.add(playerStats.getUUID()); }
+    public void addTrusted(String trustedPlayerUUID) { trustedUUIDs.add(trustedPlayerUUID); }
 
     public void removeTrusted(PlayerStats playerStats) { trustedUUIDs.remove(playerStats.getUUID()); }
```

Comment:
For code readability, there's no need for the removeTrusted that uses a PlayerStats object if we have a String UUID since they do the same thing. so I would remove the PlayerStats removeTrusted and addTrusted. Then fix any broken calls that used it elsewhere (you'd know by building the plugin)

Reply:
cleaned up